bundle/config/root.go

```diff
@@ -457,3 +457,14 @@ func validateVariableOverrides(root, target dyn.Value) (err error) {
 
 	return nil
 }
+
+// Best effort to get the location of configuration value at the specified path.
+// This function is useful to annotate error messages with the location, because
+// we don't want to fail with a different error message if we cannot retrieve the location.
+func (r *Root) TryLocation(path string) dyn.Location {
```

Comment:
I'd call it `GetLocation` as try implies that it might file but it does not return an error which is confusing